system/settings/gbm.xml

```diff
@@ -47,6 +47,11 @@
         <setting id="videoscreen.screen">
           <visible>false</visible>
         </setting>
+	<setting id="videoscreen.hwscalingfilter" type="boolean" label="39120" help="39121">
+	  <level>3</level>
+	  <default>false</default>
```

Comment:
This is a cool feature. Why default it to off?

Reply:
I agree with you, let me know if you want me to make it default ON.

Reply:
Big yes

Reply:
Thank you @garbear , I have done the changes and updated the PR, kindly let me know if anything else needs to be done from my side to take it forward.